sdk/nodejs/output.ts

```diff
@@ -789,7 +786,8 @@ type NonFunctionPropertyNames<T> = { [K in keyof T]: T[K] extends Function ? nev
 // Lift up all the non-function properties.  If it was optional before, keep it optional after.
 // If it's require before, keep it required afterwards.
 export type LiftedObject<T, K extends keyof T> = {
-    [P in K]: Output<T[P]>
+    [P in K]: T[P] extends OutputInstance<infer T1> ? Output<T1> :
+              T[P] extends Promise<infer T2> ? Output<T2> : Output<T[P]>
```

Comment:
needed to properly encode that a lifted property that is already an Output/Promise doesn't get wrapped in *another* layer of `Output`.  i.e. you should not get back `Output<Output<string>>`